devel/libbsd/Portfile

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
```

Comment:
This portgroup isn't used.
```suggestion
```

Reply:
Granted. Will use `PortGroup gitlab 1.0` instead.